desktop/apps/buy_art/test/fixture.json

```diff
@@ -42,7 +42,7 @@
         {
           "image": "http://files.artsy.net/images/data.svg",
           "title": "Comprehensive Data",
-          "content": "Consule our database of artists, exhibitors, auction results, and market news"
+          "content": "Consult our database of artists, exhibitors, auction results, and market news"
```

Comment:
It's good to update the fixture here. I believe you can also updated it via the `/collect-art/edit` UI now that we have that working. That's pending we made the appropriate file name change on S3.